joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/ExternalCommand.scala

```diff
@@ -10,7 +10,7 @@ object ExternalCommand {
 
   private val IS_WIN: Boolean = scala.util.Properties.isWin
 
-  private val shellPrefix: Seq[String] = if (IS_WIN) "cmd" :: "/c" :: Nil else "sh" :: "-c" :: Nil
```

Comment:
Why is that? This change will affect all frontends running external programs so I just want to understand that :-)

Reply:
Oops! Committed this by mistake, will push a fix commit. Thanks.